dataactbroker/handlers/generation_handler.py

```diff
@@ -72,6 +72,9 @@ def generate_file(submission, file_type, start, end, agency_type, file_format):
     try:
         if file_type in ['D1', 'D2']:
             generation_helper.start_d_generation(job, start, end, agency_type, file_format=file_format)
+        elif file_type == 'A':
+            agency_code = submission.frec_code if submission.frec_code else submission.cgac_code
```

Comment:
Could use Python's coalesce
```suggestion
            agency_code = submission.frec_code or submission.cgac_code
```